modules/openapi-generator/src/main/resources/JavaSpring/paramDoc.mustache

```diff
@@ -0,0 +1 @@
+{{#oas3}}@Parameter(name = "{{{baseName}}}", description = "{{{description}}}"{{#required}}, required = true{{/required}}, schema = @Schema(description = ""{{#allowableValues}}, {{> allowableValues }}{{/allowableValues}}{{^isContainer}}{{#defaultValue}}, defaultValue = "{{{.}}}"{{/defaultValue}}{{/isContainer}})){{/oas3}}{{^oas3}}@ApiParam(value = "{{{description}}}"{{#required}}, required = true{{/required}}{{#allowableValues}}, {{> allowableValues }}{{/allowableValues}}{{^isContainer}}{{#defaultValue}}, defaultValue = "{{{.}}}"{{/defaultValue}}{{/isContainer}}){{/oas3}}
```

Comment:
style: spacing around `{{> allowableValues }}` is inconsistent with other files (e.g. `{{>dateTimeParam}}`

Reply:
I used the spacing to match the style elsewhere in the project. Just search for the exact string.